GOSS_tests_for_sbps.md

```diff
@@ -0,0 +1,48 @@
+# Assumptions before running this test: 
+
+Ansible scripts are run successfully which configures iSCSI on selected worker nodes and label them as 'iscsi=sbps'. Goss tests use this label as identification of nodes on which this test has to run. 

```

Comment:
"and label them as" -> "and **labels** them **with**"

Reply:
done